src/emulator/controller.ts

```diff
@@ -330,7 +329,7 @@ export async function startAll(options: Options, showUI: boolean = true): Promis
   const hubLogger = EmulatorLogger.forEmulator(Emulators.HUB);
   hubLogger.logLabeled("BULLET", "emulators", `Starting emulators: ${targets.join(", ")}`);
 
-  const projectId: string = needProjectId(options);
+  const projectId: string = getProjectId(options) || ""; // TODO: Next breaking change, consider making this fall back to demo project.
```

Comment:
I think we should strongly consider giving this a sane default, but I'm afraid to change the behavior at all given our recent run ins with Hyrum's law.